SPECS/uclibc-ng/uclibc-ng.spec

```diff
@@ -3,15 +3,14 @@
 %global debug_package %{nil}
 Summary:        C library for embedded Linux
 Name:           uclibc-ng
-Version:        1.0.37
-Release:        2%{?dist}
+Version:        1.0.41
+Release:        1%{?dist}
 License:        LGPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://www.uclibc.org/
```

Comment:
Nit: While you're here, can you change the URL to `https://uclibc-ng.org/`?

Reply:
definitely.  Fixed.